build.gradle

```diff
@@ -143,10 +143,10 @@ processResources {
 	dependsOn verifyBlade
 }
 
-def intellijPortalToken = credentials.intellijPortalToken ?: ""
+def intellijApiKey = credentials.intellijApiKey ?: ""
```

Comment:
hey Greg, I saw you change the variable name, how did you get this name? or just rename it with new one?

Reply:
Unfortunately, I ran the command ./gradlew clean publishPlugin -PintellijApiKey="xxxxx" but failed. Any ideas about this?